src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -332,27 +332,16 @@ MenuGame::MenuGame(Steam::steam_manager& mSteamManager,
     setIndex(randomLevel);
 
     // Setup for the loading menu
-    static const std::vector<std::string> randomTips{
-        "HOLDING SHIFT WHILE CHANGING PACK\nSKIPS THE SWITCH ANIMATION\n",
-        "REMEMBER TO TAKE BREAKS\nOPEN HEXAGON IS AN INTENSE GAME\n",
-        "EXPERIMENT USING SWAP\nIT MAY SAVE YOUR LIFE\n",
-        "IF A LEVEL IS TOO CHALLENGING\nPRACTICE IT AT A LOWER DIFFICULTY\n"};
-
-    std::string temp;
-    for(char c : randomTips[ssvu::getRndI(0, randomTips.size())])
-    {
-        if(c == '\n')
-        {
-            randomTip.emplace_back(temp);
-            temp.clear();
-        }
-        else
-        {
-            temp += c;
-        }
-    }
+    static const std::string tips[][2]{
+        {"HOLDING SHIFT WHILE CHANGING PACK", "SKIPS THE SWITCH ANIMATION"},
+        {"REMEMBER TO TAKE BREAKS", "OPEN HEXAGON IS AN INTENSE GAME"},
+        {"EXPERIMENT USING SWAP", "IT MAY SAVE YOUR LIFE"},
+        {"IF A LEVEL IS TOO CHALLENGING", "PRACTICE IT AT A LOWER DIFFICULTY"}
+    };
```

Comment:
```suggestion
    static constexpr std::array<std::string_view, 2> tips{
        {"HOLDING SHIFT WHILE CHANGING PACK", "SKIPS THE SWITCH ANIMATION"},
        {"REMEMBER TO TAKE BREAKS", "OPEN HEXAGON IS AN INTENSE GAME"},
        {"EXPERIMENT USING SWAP", "IT MAY SAVE YOUR LIFE"},
        {"IF A LEVEL IS TOO CHALLENGING", "PRACTICE IT AT A LOWER DIFFICULTY"}
    };
```